### PR TITLE
german translation

### PR DESCRIPTION
**Pull request by rschnapka**
initial german translation, hr_language not fully done (yet)

### CODE REVIEW

File: hr_language/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+
+#. module: hr_language
+#: selection:hr.language,name:0
+msgid "Japanese / 日本語"
+msgstr ""
+
+#. module: hr_language
+#: selection:hr.language,name:0
+msgid "Russian / русский язык"
+msgstr ""
```
Comment (pedrobaeza) [inline hr_language/i18n/de.po]: You cannot let this empty. At least put the same as in English.

**Comment by pedrobaeza** [comment]
Please put also the updated pot files if they are missing.

**Comment by rschnapka** [comment]
ok... thought an empty translation would autom. take untranslated term. But fine - will do :-)

**Comment by rschnapka** [comment]
properly added full translation terms now...

**Comment by pedrobaeza** [comment]
Can you please address the other comment?

**Comment by rschnapka** [comment]
You probably mean the completely missing i18n folders? I'll look into creating those as needed - are they generally needed, even if no terms are translatable in the module?

**Comment by pedrobaeza** [comment]
I mean the pot files - translation templates - (if any is missing).

Thanks for your contributions.

**Comment by rschnapka** [comment]
each and every de.po file was derived from a pot file - so in fact there should be none missing. The only possible open issue in this context might have been missing translation (i18n-folder) of a module as such ... that's why i was asking...
In this context every module that had a *.pot file now also has a de.po file. 
AND If there is no pot-file then there is also no i18n folder that is missing a pot.

**Comment by pedrobaeza** [comment]
Ah, OK. Then it's :+1: for me

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**